docs/2-creating-an-extension-to-allow-custom-tags.md

```diff
@@ -135,3 +135,14 @@ $sanitizer = $builder->build([
     'extensions' => ['basic', 'list', 'my-tag'],
 });
 ```
+
+You can also use `HtmlSanitizer\SanitizerBuilder::withDefaultConfig()` to get a `SanitizerBuilder` with all the core extensions:
+
+```php
+$builder = HtmlSanitizer\SanitizerBuilder::withDefaultConfig();
```

Comment:
I would prefer to use `createDefault` to be consistent with `Sanitizer::create`, do you mind to update this name :) ?

Reply:
Done.